RasimKhusaenov/my_own_blog
# Styles are not recompiled automatically

### Summary

- SCSS files moved to `app/javascript`
- `application.scss` moved to `app/javascript/packs`
- Changed stylesheet helper in `application.html.slim`

### How it works

Screenshots/screencasts of the pull request introduced functionality.

### Test plan

List of steps to manually test introduced functionality:

- Go to http://localhost:3000
- Change style in some file (ex. `flex-direction: column;` -> `flex-direction: row;`)
- Update page
- Make sure the style is applied

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
- Resolves #66 Issue
- GitHub Pull Request #64
